src/content_script.css

```diff
@@ -125,63 +138,82 @@
 	opacity: 1;
 }
 
-.fbc-badge-prompt {
-	display: none;
-	width: 345px;
-	position: absolute;
-	left: calc( 100% + 20px);
-	top: 50%;
-  transform: translateY(-50%);
+.fbc-container {
+	overflow: hidden;
+}
+
+.fbc-wrapper{
+	position: fixed;
+	height: 100%;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	width: 100%;
+	z-index: 999999999999;
+	background-color: rgba(0, 0, 0, 0.1);
+	/* animation: 0.2s ease forwards; */
+}
+
+.fbc-wrapper iframe {
 	color: var(--black);
 	box-shadow: 0 5px 12px 0 rgba(0,0,0,0.2);
 	border: 1px solid rgba(0,0,0,0.2);
-	padding: 0 0 40px;
+	background: var(--white);
 }
 
-.fbc-badge-prompt:after {
-	content: " ";
+.fbc-iframe-chevron {
```

Comment:
suggestion: On Relay, I migrated away from the CSS triangle, and instead, used an [inline SVG shape](https://github.com/mozilla/fx-private-relay-add-on/blame/9da1e84c5f2140a5cd3b74b3edad31fbb95f5723/src/css/relay.css#L62-L71).